configs/benchmark/pytorch/pytorchxla_torchbench_config.py

```diff
@@ -119,3 +107,132 @@ def get_torchbench_config(
       task_gcp_config=job_gcp_config,
       task_metric_config=job_metric_config,
   )
+
+
+# Below is the setup for torchbench GPU run.
+def set_up_torchbench_gpu(model_name: str = "") -> Tuple[str]:
+  """Common set up for TorchBench."""
+  # TODO(piz): There is issue with driver install through fabric.
```

Comment:
The issue is installing GPU driver through fabric. The installation will eventually fail due to cuda is in use. I tried many ways to kill the cuda related process before the installation but still doesn't work. One assumption I think is that fabric connects with the host and enables the x server, which prevents the driver installation since the x server is using the cuda.

Reply:
Fixed based on comment: https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/45#discussion_r1424694127.